float64 enemy_x
float64 enemy_y
